### PR TITLE
update logs with "cannot render instance groups" instead of "cannot render nodes" in validate_cluster.go

### PR DESCRIPTION
**Pull request by haojue**
update logs with "cannot render instance groups" instead of "cannot render nodes"

### CODE REVIEW

File: cmd/kops/validate_cluster.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ func RunValidateCluster(ctx context.Context, f *util.Factory, out io.Writer, opt
 					time.Sleep(pollInterval)
 					continue
 				} else {
-					return nil, fmt.Errorf("cluster passed validation %d consecutive times", consecutive)
+					return nil, fmt.Errorf("cluster passed validation %d consecutive times, failed because of negative options.wait %d", consecutive, options.wait)
```
Comment (johngmyers) [inline cmd/kops/validate_cluster.go:225]: This message is inaccurate. Options.wait is not necessarily negative. 

The lack of a wait option is not a cause of the cluster's failure to pass validation.

**Review by johngmyers — CHANGES_REQUESTED** [review]

File: cmd/kops/validate_cluster.go (hunk 1 above)
Comment (haojue) [reply cmd/kops/validate_cluster.go:225]: Hi @johngmyers , you mean when it fails, Options.wait is not necessarily negative.?  I just noticed ` if options.wait > 0 {` , so what's your recommended error message for it? thanks

File: cmd/kops/validate_cluster.go [hunk 2]
```diff
@@ -264,7 +264,7 @@ func validateClusterOutputTable(result *validation.ValidationCluster, cluster *k
 	fmt.Fprintln(out, "INSTANCE GROUPS")
 	err := t.Render(instanceGroups, out, "NAME", "ROLE", "MACHINETYPE", "MIN", "MAX", "SUBNETS")
 	if err != nil {
-		return fmt.Errorf("cannot render nodes for %q: %v", cluster.Name, err)
+		return fmt.Errorf("cannot render instancegroups for %q: %v", cluster.Name, err)
```
Comment (haojue) [inline cmd/kops/validate_cluster.go:267]: Hi @johngmyers , this is ok to you?

Comment (johngmyers) [reply cmd/kops/validate_cluster.go:267]:
```suggestion
		return fmt.Errorf("cannot render instance groups for %q: %w", cluster.Name, err)
```

File: cmd/kops/validate_cluster.go (hunk 1 above)
Comment (hakman) [reply cmd/kops/validate_cluster.go:225]: > so what's your recommended error message for it? thanks

I is recommended to leave it as it is, without changes.

**Comment by haojue** [comment]
Hi @johngmyers , thank you for your review comments, I updated per your comments.

**Comment by johngmyers** [comment]
@haojue please change the commit and PR descriptions to match the remaining change.

**Comment by haojue** [comment]
Hi @johngmyers, I updated the description and the commit message

**Review by johngmyers — APPROVED** [review]
Thanks!

**Pull request merged**